templates/minimal/README.md

```diff
@@ -11,30 +11,54 @@
 
 </div>
 
-* 🤏 This template is a minimal (in terms of complexity and the number of components)
+## Table of Contents
+
+- [Intro](#intro)
+
+- [Template Structure](#template-structure)
+
+- [Getting Started](#getting-started)
+
+- [Starting a Minimal Template Chain](#starting-a-minimal-template-chain)
+
+  - [Omni Node](#omni-node)
+  - [Minimal Template Node](#minimal-template-node)
+  - [Zombienet with Omni Node](#zombienet-with-omni-node)
+  - [Zombienet with Minimal Template Node](#zombienet-with-minimal-template-node)
+  - [Connect with the Polkadot-JS Apps Front-End](#connect-with-the-polkadot-js-apps-front-end)
+  - [Takeaways](#takeaways)
+
+- [Contributing](#contributing)
+
+- [Getting Help](#getting-help)
+
+## Intro
+
+- 🤏 This template is a minimal (in terms of complexity and the number of components)
 template for building a blockchain node.
 
-* 🔧 Its runtime is configured with a single custom pallet as a starting point, and a handful of ready-made pallets
+- 🔧 Its runtime is configured with a single custom pallet as a starting point, and a handful of ready-made pallets
 such as a [Balances pallet](https://paritytech.github.io/polkadot-sdk/master/pallet_balances/index.html).
```

Comment:
I don't have a strong opinion, but pointing out our lack fo strategy abou linking to `docs.rs` vs our master-hosted version. 

Once the #6191 is solved, I would advocate to only link to our internal version.